### DISCUSSION TITLE
Update folder structure

### DISCUSSION THREAD

**Discussion by xlc** [paritytech/substrate-open-working-groups#6] [General]
The folder structure for the SOWGs are a bit confusing to me.

There are four stages for a SOWG, Draft, Activation, Maintain, Closed
Drafts are on root folder.
Resources are in src folder with the SOWG number as name. What happen if multiple images are needed?
Where will the activated SOWG located? root folder as well?
Closed one are in archive folder.
And there is a draft folder.

I don't like the idea of moving files around, it invalidates direct link, make it hard to search for a particular SOWG.

I propose to just have a folder of `sowg` that have all the SOWG files. Those files should include the status and there should be a readme file listing all the SWOGs and their status as well.

**Comment by BenWhiteJam** [comment]
> The folder structure for the SOWGs are a bit confusing to me.
> 
> There are four stages for a SOWG, Draft, Activation, Maintain, Closed
> Drafts are on root folder.
> Resources are in src folder with the SOWG number as name. What happen if multiple images are needed?

The idea was to have images related by working group number prefix and separated by name
|- src
|-- 1-token-architecture
|-- 1-fn-overview
|-- 2-identity-overview
|-- ...

> Where will the activated SOWG located? root folder as well?
> Closed one are in archive folder.
> And there is a draft folder.
> 
> I don't like the idea of moving files around, it invalidates direct link, make it hard to search for a particular SOWG.

See your point of invalidating the direct link and that makes sense to me, will update accord to the answer to the next question.

> I propose to just have a folder of `sowg` that have all the SOWG files. Those files should include the status and there should be a readme file listing all the SWOGs and their status as well.

Okay, so it would be something like:
|- SOWG
|-- 1-polkadot-token-standard.md
|-- 2-DID..
|- readme.md (with sections: `draft`, `active`, `closed`,... linking to the actual md`s)
